R/get_predict_model.R

```diff
@@ -24,10 +24,12 @@ get_predict_model <- function(predict_model, model) {
     if (native_func_available) {
       predict_model <- get(paste0("predict_model.", model_class0))
     } else {
-      stop(
-        "You passed a model to explain() which is not natively supported, and did not supply the 'predict_model' ",
-        "function to explain().\n",
-        "See ?shapr::explain or the vignette for more information on how to run shapr with custom models."
+      cli::cli_abort(
+        paste0(
+          "You passed a model to explain() which is not natively supported, and did not supply the 'predict_model' ",
+          "function to explain().\n",
+          "See ?shapr::explain or the vignette for more information on how to run shapr with custom models."
```

Comment:
```suggestion
          "See `?shapr::explain` or the vignette for more information on how to run shapr with custom models."
```

I agree that here it would have been nice to have it be a clickable link.